javatests/arcs/android/integration/policy/scenarios.arcs

```diff
@@ -12,6 +12,29 @@ meta
 
 import './particles.arcs'
 
+/// Scenario: No data is written
+@egressType('GeneralEgress')
```

Comment:
Ingress restricting should allow the fields mentioned in the policy to be persisted. However, due to the fact that we are not plumbing the policy to `recipe2plan`, that does not happen. 

Either remove the policy or add a reference to bug b/162992168

Reply:
I decided to do both. How does this look?